components/input-box/inputBox.css

```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap");
+
+* {
+  top: 0;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
this can put in a common file ,similarly for colors which are not specific to this file. change it later when you find time.